marimbabot_vision/scripts/data_augmentation.py

```diff
@@ -0,0 +1,52 @@
+from generate_data import NUM_SAMPLES, NUM_WORKER, OUTPUT_DIR
+from multiprocessing import Pool
+from PIL import Image
+
+import tqdm
+import shutil
+import albumentations as A
+import numpy as np
+import cv2
+
+def apply_transforms(img, transforms = [A.RGBShift()]):
```

Comment:
Instantiating objects in a conditional argument is a bad practice in python.

Reply:
It does not really matter here but they are only initialized once and this might lead to weird behavior.